ao3_get_fanfics.py

```diff
@@ -189,24 +195,48 @@ def write_fic_to_csv(fic_id, only_first_chap, writer, errorwriter, header_info='
 		visible_kudos = get_kudos(soup.find('p', class_='kudos'))
 		hidden_kudos = get_kudos(soup.find('span', class_='kudos_expanded hidden'))
 		all_kudos = visible_kudos + hidden_kudos
-		
-		#get bookmarks
-		bookmark_url = 'http://archiveofourown.org/works/'+str(fic_id)+'/bookmarks'
-		all_bookmarks = get_bookmarks(bookmark_url, header_info)
+		stats2 = ' '
+		stats2 = stats2.join(stats)
+		sep = ' '
+		stripped = stats2.split(sep, 1)[0]
```

Comment:
stats2 and stripped aren't necessary -- all the info you want is already in the stats array.  you can grab the language just by saying stats[0]